card/lib/card/loader.rb

```diff
@@ -15,12 +15,53 @@ def paths
   end
 
   module Loader
-
     class << self
       def load_mods
         load_set_patterns
         load_formats
         load_sets
+
+        if ENV['RAILS_ENV'] == 'development'
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.